ONSdigital/log.go
# Implementation of log.go

### What

Implemented logger for go which matches logging standards.

### How to review

* Check README makes sense
* Check godoc makes sense
* Check tests are useful, pass, and don't have any obvious gaps
* Check the output matches the spec
* Review the implementation code
* Have a look at ONSdigital/florence#330 to see how its used
* Try using it!

### Who can review

Anyone except @ian-kent
